pgd_splicer/tests.py

```diff
@@ -329,3 +329,49 @@ def test_crosscheck_cullpdb(self):
 
         good_out = MonkeyPatch.sitefile('cullpdb_crosscheck.txt')
         self.assertEqual(test_out, file(good_out).read())
+
+
+class ProcessPDBTask(TestCase):
+
+    def test_import_pdb(self):
+
+        # Choose the first protein in the selection file.
+        with open(MonkeyPatch.sitefile('cullpdb_selection.txt'), 'r') as f:
+            line = f.readline().split(' ')
+
+        # Build pdbs variable.
+        # NB: structure taken from process_args() in ProcessPDBTask.main()
+        pdbs = [
```

Comment:
We should get rid of this in favor of passing the `line` to `task.work(...)`

This would require some back-end changes but I think in the long run it's worth it to improve code instead of covering it with duct-tape and weird variations of off-white paint.